From 36fd4f214f9ba74aaf0e5fb3e4ba271b946a1550 Mon Sep 17 00:00:00 2001
From: Thomas Graf <tgraf@noironetworks.com>
Date: Wed, 26 Nov 2014 15:52:31 +0100
Subject: [PATCH] datapath: Use ccflags-y instead of deprecated EXTRA_CFLAGS

This allows users to pass in additional compiler flags through the
environment variable EXTRA_CFLAGS, e.g.

   make EXTRA_CFLAGS=-Wno-error=foo V=1

Reported-by: Alexandru Ardelean <ardeleanalex@gmail.com>
Signed-off-by: Thomas Graf <tgraf@noironetworks.com>
Acked-by: Pravin B Shelar <pshelar@nicira.com>
---
 datapath/linux/Kbuild.in | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/datapath/linux/Kbuild.in b/datapath/linux/Kbuild.in
index 6f6f65f..cb98c11 100644
--- a/datapath/linux/Kbuild.in
+++ b/datapath/linux/Kbuild.in
@@ -7,11 +7,11 @@ export VERSION = @VERSION@
 include $(srcdir)/../Modules.mk
 include $(srcdir)/Modules.mk
 
-EXTRA_CFLAGS := -DVERSION=\"$(VERSION)\"
-EXTRA_CFLAGS += -I$(srcdir)/..
-EXTRA_CFLAGS += -I$(builddir)/..
-EXTRA_CFLAGS += -g
-EXTRA_CFLAGS += -include $(builddir)/kcompat.h
+ccflags-y := -DVERSION=\"$(VERSION)\"
+ccflags-y += -I$(srcdir)/..
+ccflags-y += -I$(builddir)/..
+ccflags-y += -g
+ccflags-y += -include $(builddir)/kcompat.h
 
 # These include directories have to go before -I$(KSRC)/include.
 # NOSTDINC_FLAGS just happens to be a variable that goes in the
-- 
2.1.2

